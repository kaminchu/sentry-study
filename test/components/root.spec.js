// @flow
import React from "react";
import Root from "../../src/js/components/root";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

describe("components/root", () => {
  test("snapshot test", () => {
    const component = renderer.create(
      <Root />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  test("dom test", () => {
    const root = shallow(<Root/>);
    expect(root.text()).toEqual("Hello, World!");
  });
});