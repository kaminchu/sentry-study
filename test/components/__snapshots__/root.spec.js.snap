// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/root snapshot test 1`] = `
<div>
  Hello, World!
</div>
`;
